main.go

```diff
@@ -98,6 +104,18 @@ func mainWithError() (err error) {
 		}
 	}
 
+	daemonCommand := newCommand.DaemonCommand().CobraCommand()
+
+	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.Address, "", "Address used to connect to Kubernetes.")
+	daemonCommand.PersistentFlags().Bool(f.Service.Kubernetes.InCluster, true, "Whether to use the in-cluster config to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CAFile, "", "Certificate authority file path to use to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
+
+	daemonCommand.PersistentFlags().String(f.Service.Secrets.Namespace, "default", "Namespace to store secrets in.")
+	daemonCommand.PersistentFlags().String(f.Service.Secrets.NameFormat, "credential-%s", "Format to name secrets with.")
+	daemonCommand.PersistentFlags().Int(f.Service.Secrets.IDLength, 6, "Length of credential IDs.")
```

Comment:
Is there a real reason for these three guys be configurable?

Do we go with the `default` namespace? Should also the default secret be there?

Reply:
No real reason, it just felt right to me :D

I was under the impression we were going with the `default` namespace, to match CRs in general. If we go with `default`, the default secret should be there too.